data/npc/scripts/Amber.lua

```diff
@@ -33,88 +32,553 @@ keywordHandler:addKeyword(
-	prisonerKeyword:addChildKeyword({"yes"}, StdModule.say, {npcHandler = npcHandler, text = "It\"s \"mok\" in Orcish. I tell you more if you bring me some {food}.", reset = true})
-	prisonerKeyword:addChildKeyword({"no"}, StdModule.say, {npcHandler = npcHandler, text = "In Orcish that\"s \"burp\". I tell you more if you bring me some {food}.", reset = true})
+keywordHandler:addKeyword(
+	{"orc"},
+	StdModule.say,
+	{
+		npcHandler = npcHandler,
+		text = "Not the nicest guys you can encounter. I had some clashes with them and finally ended up being their {prisoner} for a few months."
+	})
+
+local prisonerKeyword = keywordHandler:addKeyword(
+	{"prisoner"},
+	StdModule.say,
+	{
+		npcHandler = npcHandler,
+		text = "I speak some Orcish words, not many though, just a few basics like '{yes}' and '{no}'."
+	})
+
+prisonerKeyword:addChildKeyword(
+	{"yes"},
+	StdModule.say,
+	{
+		npcHandler = npcHandler,
+		text = "It's 'mok' in Orcish. I tell you more if you bring me some {food}.",
+		reset = true
+	})
+
+prisonerKeyword:addChildKeyword(
+	{"no"},
+	StdModule.say,
+	{
+		npcHandler = npcHandler,
+		text = "In Orcish that's 'burp'. I tell you more if you bring me some {food}.",
+		reset = true
+	})
+
 keywordHandler:addAliasKeyword({"language"})
 
 -- Food (Salmon)
-keywordHandler:addKeyword({"food"}, StdModule.say, {npcHandler = npcHandler, text = "My favourite dish is {salmon}. Oh please, bring me some."})
-local salmonKeyword = keywordHandler:addKeyword({"salmon"}, StdModule.say, {npcHandler = npcHandler, text = "Yeah! If you give me some salmon, I\"ll tell you another Orcish word. Okay?"})
-	salmonKeyword:addChildKeyword({"yes"}, StdModule.say, {npcHandler = npcHandler, text = "Thank you. Orcs call arrows \"pixo\".", reset = true},
-		function(player) return player:getItemCount(2668) > 0 end,
-		function(player) player:removeItem(2668, 1) end
-	)
-	salmonKeyword:addChildKeyword({"yes"}, StdModule.say, {npcHandler = npcHandler, text = "You don\"t have any salmon!", reset = true})
-	salmonKeyword:addChildKeyword({""}, StdModule.say, {npcHandler = npcHandler, text = "Ok, then I won\"t tell you another Orcish word.", reset = true})
+keywordHandler:addKeyword(
+	{"food"},
+	StdModule.say,
+	{
+		npcHandler = npcHandler,
+		text = "My favourite dish is {salmon}. Oh please, bring me some."
+	})
+
+local salmonKeyword = keywordHandler:addKeyword(
+	{"salmon"},
+	StdModule.say,
+	{
+		npcHandler = npcHandler,
+		text = "Yeah! If you give me some salmon, I'll tell you another Orcish word. Okay?"
+	})
+
+salmonKeyword:addChildKeyword(
+	{"yes"},
+	StdModule.say,
+	{
+		npcHandler = npcHandler,
+		text = "Thank you. Orcs call arrows 'pixo'.",
+		reset = true
+	},
+	function(player) return player:getItemCount(2668) > 0 end,
+	function(player) player:removeItem(2668, 1) end
+)
+
+salmonKeyword:addChildKeyword(
+	{"yes"}, D
```

Comment:
Remove `D` from this line, it's breaking the code.

```suggestion
	{"yes"},
```